include/items.php

```diff
@@ -401,17 +401,17 @@ function drop_item($id)
 		}
 		// Now check how the user responded to the confirmation query
 		if (!empty($_REQUEST['canceled'])) {
-			goaway(System::baseUrl() . '/' . $_SESSION['return_url']);
+			goaway('/item/drop/' . $id);
```

Comment:
Wait, if it is canceled, then we shouldn't redirect to the deletion screen. Better go to `/display`.

Reply:
How I reproduce the `$_REQUEST['canceled']` situation ? frio and vier are canceling via JS